cpp/src/barretenberg/stdlib/primitives/bigfield/bigfield.test.cpp

```diff
@@ -712,6 +712,42 @@ template <typename Composer> class stdlib_bigfield : public testing::Test {
         EXPECT_EQ(result, true);
     }
 
+    static void test_assert_less_than_success()
```

Comment:
Please create a test that checks specifically for the edge case a<a. I think that the assert_less_than function will allow it, since we do a-a and check that it can be range constrained.